pkg/apis/probe/query.go

```diff
@@ -0,0 +1,115 @@
+package probe
+
+const (
+	ListProbeQuery = `query ListProbes($projectID: ID!, $probeNames: [ID!], $filter: ProbeFilterInput) {
+		listProbes(projectID: $projectID, probeNames: $probeNames, filter: $filter) {
+		  name
+		  type
+		  createdAt
+		  createdBy{
+			username
+		  }
+		}
+	  }
+	`
+	GetProbeQuery = `query getProbe($projectID: ID!, $probeName: ID!) {
+		getProbe(projectID: $projectID, probeName: $probeName) {
+		  name
+		  description
+		  type
+		  infrastructureType
+		  kubernetesHTTPProperties{
+			probeTimeout
+			interval
+			retry
```

Comment:
Are we using all the properties? If not, then we can consider removing them from query